### ISSUE TITLE
Remove misleading hover Cypress tests

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#588]
**Is your feature request related to a problem? Please describe.**
We currently have a Cypress test for checking the hover styles of the brand within the header. This is actually a bug as the test only passing as the previous test Focuses the element and does not remove focus (`element.blur()`) at the end of the test. Because of this the hover test appears to pass but this is actually due to the focus state adding the same styling as the hover.

**Describe the solution you'd like**
Remove the current hover tests until Cypress adds a reliable way to test for the css pseudo selectors such as hover. 

**Describe alternatives you've considered**
We could use classes to add and remove the hover styles using client side JS, then we could test the hover styling. 

**Additional context**
https://github.com/cypress-io/cypress/issues/10#issuecomment-118107915
TLDR; `it is not possible to trigger the CSS pseudo selectors via JavaScript directly`
